web/src/models/requests/authRequests.ts

```diff
@@ -0,0 +1,12 @@
+import { Role } from "../role"
+
+export interface LoginRequest {
+	username: string
+	password: string
+}
+
+export interface CreateAccountRequest {
+	username: string
+	password: string
+	role: Role
```

Comment:
нужно передавать роль через id
```suggestion
	roleId: number
```

Reply:
согласен, тогда когда сделаем функционал ролей, тут это уберется